infra/gcp/terraform/k8s-infra-ii-sandbox/main.tf

```diff
@@ -69,3 +69,9 @@ resource "google_bigquery_dataset_iam_member" "bq_data_transfer_binding" {
   role       = "roles/bigquery.admin"
   member     = "serviceAccount:${data.google_service_account.bq_data_transfer.email}"
 }
+
+resource "google_service_account" "asn_etl" {
```

Comment:
@BobyMCbobs early review; Can we have a similar service account on https://github.com/kubernetes/k8s.io/tree/main/infra/gcp/terraform/k8s-infra-public-pii ?

Reply:
Thank you @ameukam for taking a look at this PR.

What would the ServiceAccount be called and used for?

Reply:
You can keep the same name. I would to see any new change done in this project mirrored in `k8s-infra-public-pii`.

Reply:
Right! That will make it easy for having it moved over to production when fully ready.

Reply:
OK, made the change in a556ada1